README.md

```diff
@@ -1,13 +1,131 @@
-# Salesforce ESB Inbound
+# Configuring Salesforce Inbound Endpoint
 
-The Salesforce inbound endpoint allows you to perform various Salesforce streaming data via WSO2 ESB.
+The Salesforce streaming inbound endpoint allows you to perform various Salesforce streaming data via WSO2 ESB.
 
-##Build
+1.The Salesforce streaming API receives notifications for changes to Salesforce data that match a Salesforce Object Query Language (SOQL) query you define, in a secure and scalable way. For more information see [Salesforce streaming documentation](https://developer.salesforce.com/docs/atlas.en-us.202.0.api_streaming.meta/api_streaming/quick_start_workbench.htm) .
 
-mvn clean install
+2.[Platform events](#platform-event)
 
-###How You Can Contribute
 
-You can create a third party connector and publish in WSO2 Connector Store.
+
+```
+Reliable message delivery is only available in Salesforce API version 37.0 and later.
+```
+
+## How To Use
+1.[Download the inbound endpoit jar](https://store.wso2.com/store/assets/esbconnector/details/fbb433b5-4d74-4064-84c2-e4b23c531aa2) or build the project **mvn clean install** to get the jar and add it in the <ESB-Home>/dropin directory.
```

Comment:
Either Download the inbound endpoint jar or locally build (mvn clean install) to get the connector jar